pkg/rbac/client_wrapper.go

```diff
@@ -33,43 +33,25 @@ import (
 
 const application = "content-sources"
 
-type RbacVerb string
-
-// The following constants result from the schema below
-// https://github.com/RedHatInsights/rbac-config/blob/master/schemas/permissions.schema
-const (
-	RbacVerbAny       RbacVerb = "*"
-	RbacVerbRead      RbacVerb = "read"
-	RbacVerbWrite     RbacVerb = "write"
-	RbacVerbCreate    RbacVerb = "create"
-	RbacVerbUpdate    RbacVerb = "update"
-	RbacVerbDelete    RbacVerb = "delete"
-	RbacVerbLink      RbacVerb = "link"
-	RbacVerbUnlink    RbacVerb = "unlink"
-	RbacVerbOrder     RbacVerb = "order"
-	RbacVerbExecute   RbacVerb = "execute"
-	RbacVerbUndefined RbacVerb = ""
-)
-
-type Rbac interface {
-	Allowed(ctx context.Context, resource string, verb RbacVerb) (bool, error)
+type ClientWrapper interface {
+	Allowed(ctx context.Context, resource Resource, verb Verb) (bool, error)
```

Comment:
Initially i wanted to name this client, but it already has an 'rbac.client' import, so ClientWrapper seemed more apt